feature-list-streams/src/main/java/com/codandotv/streamplayerapp/feature_list_streams/list/presentation/widgets/StreamsCarousel.kt

```diff
@@ -34,12 +44,24 @@ fun StreamsCarousel(
 
         Spacer(modifier = Modifier.size(8.dp))
 
-        LazyRow(modifier = Modifier.fillMaxWidth()) {
-            items(contentList.size) {
-                StreamsCard(
-                    contentList[it],
-                    onNavigateDetailList,
-                )
+        LazyRow(
+            state = lazyListState,
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(140.dp)
+        ) {
+            items(
+                count = lazyPagingItems.itemCount,
+                key = lazyPagingItems.itemKey(),
+                contentType = lazyPagingItems.itemContentType()
+            ) { index ->
+                val item = lazyPagingItems[index]
+                item?.let {
```

Comment:
o que acontece se for null, ele pula? ele não preenche?

de repente no itens, tem um filterNotNul()?

Reply:
Acredito que aqui valeria a pena a gente criar issues pra tratar esse cenário, como também o cenário de erro e empty state. O que acha?